[DOC-1164] Voyager docs for 2026.2.1 (#30182)
yugabyte/yugabyte-db: docs/content/stable/yugabyte-voyager/release-notes.md

Voyager releases (starting with v2025.5.2) use the numbering format `YYYY.M.N`, where `YYYY` is the release year, `M` is the month, and `N` is the number of the release in that month.

## v2026.2.1 - February 3, 2026

{{< note title="Important: Breaking change" >}}

> This release includes breaking changes for Voyager migrations. Migrations started with earlier Voyager versions cannot be continued with this version. To proceed, either continue the migration using the same Voyager version that started it, or start a new migration with v2026.2.1.

Reviewer: ```suggestion
This release includes breaking changes for Voyager migrations. Migrations started with earlier Voyager versions cannot be continued with this version. To proceed, either continue the migration using the same Voyager version you started with, or start a new migration using v2026.2.1.
```
